utils/build-script-impl

```diff
@@ -2476,7 +2476,7 @@ for host in "${ALL_HOSTS[@]}"; do
 
                     # Handle TensorFlow Swift bindings file.
                     if [[ ! "${TENSORFLOW_SWIFT_BINDINGS}" && -d "${TENSORFLOW_SWIFT_BINDINGS_DIR}" ]] ; then
-                        TENSORFLOW_SWIFT_BINDINGS="${TENSORFLOW_SWIFT_BINDINGS_DIR}/RawOpsGenerated.swift"
```

Comment:
I don't feel it's necessary to change the build script or driver arguments. Is it?

Reply:
I believe these changes are necessary because `swift-bindings` now contains more than one Swift source file: [`EagerExecution.swift`](https://github.com/tensorflow/swift-bindings/pull/26/files#diff-8bf1351a5319726d26aa9940fb4637e6) in addition to `RawOpsGenerated.swift`.

Reply:
Yes, the reason for this is exactly what Dan said. In either case, I think this also results in more predictable behavior as you'd expect all files from the swift-bindings repo to be included.

Reply:
Not sure I get that. This PR is all about `TensorArrayProtocol` derived conformances, and it does not update checkout for tensorflow/swift-bindings. Therefore this change is not tested as part of this PR.

Reply:
I removed the change for now so it passes the tests and does not get mixed with the `TensorArrayProtocol` changes. However, it is weird that it affects the tests as it should still be working fine without updating the checkouts (i.e., with the current swift-bindings).